### PR TITLE
Spellcheck as part of CI

### PR DESCRIPTION
**Pull request by chayim**
As part of extending CI and providing a better experience working with the repository and codebase itself, I'm starting by adding some scaffolding, spelling CI.

### CODE REVIEW

**Comment by chayim** [comment]
Reminder @vladvildanov . It's in elsewhere, can we finish merging this>?

**Review by vladvildanov — APPROVED** [review]

**Comment by vladvildanov** [comment]
@chayim I'll fix broken Ci and we're good to go

**Review by vladvildanov — APPROVED** [review]

**Comment by vladvildanov** [comment]
@chayim @tillkruss Good to go

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -142,26 +142,3 @@ jobs:
...
-        uses: actions/cache@v3
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-codespell
-
-      - name: Install codespell
-        run: pip install --user 'codespell>=2.2'
-
-      - name: Search for misspellings
-        run: $(python -m site --user-base)/bin/codespell
```
Comment (tillkruss) [inline .github/workflows/linters.yml:167]: Can we keep the spellcheck in the `linters.yml`, or is there a good reason why not?

File: src/Connection/Replication/SentinelReplication.php [hunk 2]
```diff
@@ -315,31 +315,30 @@ public function getSentinelConnection()
...
-                        'host' => $sentinel[3],
-                        'port' => $sentinel[5],
-                        'role' => 'sentinel',
-                    ];
-                }
-            } catch (ConnectionException $exception) {
-                $this->sentinelConnection = null;
-
-                goto SENTINEL_QUERY;
+        try {
```
Comment (tillkruss) [inline src/Connection/Replication/SentinelReplication.php:321]: Please remove code changes from this PR that are not related to spelling.

**Review by tillkruss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**